assets/js/modules/analytics/settings/settings-view.js

```diff
@@ -45,6 +45,7 @@ export default function SettingsView() {
 	return (
 		<div className="googlesitekit-setup-module googlesitekit-setup-module--analytics">
 
+			<ErrorNotice />
```

Comment:
You moved the `ErrorNotice` component around recently to a void duplication (see e.g. https://github.com/google/site-kit-wp/pull/1454/commits/0e19750e7b5b1c794d92a40be29c4c654d7ef37e) - just want to make sure adding this here doesn't re-introduce such duplication potentially?

Reply:
Yeah, I think it probably does. I've updated it to prevent that 👍